tensorflow/python/layers/base.py

```diff
@@ -696,7 +696,7 @@ def _unique_layer_name(name):
 
   Example:
 
-  ```
+  ```python
     >>> _unique_layer_name('dense')
```

Comment:
In the rest of the examples we have no carets, and the result in an inline comment.

You could do the same thing here.